src/components/tasks/card/index.tsx

```diff
@@ -105,16 +130,8 @@ const Card: FC<Props> = ({
             width={iconWidth}
             height={iconHeight}
           />
-          <span className={classNames.cardSpecialFont}>Due Date</span>
-          <span
-            className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'endsOn')}
-            role="button"
-            tabIndex={0}
-          >
-            {fromNowEndsOn}
-          </span>
+          <span className={classNames.cardSpecialFont}>Due Date</span>      
+         {renderDate(fromNowEndsOn,shouldEdit)}
```

Comment:
Wouldn't it be better to create a component passing the props?

Reply:
@harshith-venkatesh , this is only 16 line code. 
Do you think, we should create component for this?

Reply:
Ok I agree with you
Couldn't we have gone with ternary condition here instead of function? As I observe based on shouldEdit we are showing snippet right?